src/pages/Campaign/CampaignButtonWithOptions.tsx

```diff
@@ -196,6 +196,7 @@ export default function CampaignButtonWithOptions({
 
   return (
     <StyledPrimaryButton
+      id="swap-now-button"
```

Comment:
Changes id of these buttons can affect data tracked for Product team

Reply:
This is asked by Product team